src/Plane.js

```diff
@@ -1,10 +1,21 @@
 class Plane { 
     flightNumber;
+    takeOff;
+    notInThisAirport;
```

Comment:
notInThisAirport possible should not have been a property of Plane. Maybe location (what airport, inAir etc)